From 57ce33a45bf77033737cd8a7a507410326d1d4f1 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 1 Oct 2024 16:01:02 +0200
Subject: [PATCH 1274/1695] plat-stm32mp2: conf: default enable
 CFG_DT_CACHED_NODE_INFO

Explicitly default enable CFG_DT_CACHED_NODE_INFO on plat-stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 0742d562687197f1078c92b3e1c8b76bd7812b34 plat-stm32mp2: conf: default enable CFG_DT_CACHED_NODE_INFO]
Change-Id: I4cd1bf1da1a243f9e2d67bba635f9b52d6826b1a
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/408098
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 7e1afa69e..9295869a8 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -334,3 +334,7 @@ endif
 
 # Support for STM32MP OSTL legacy SCMI messages
 CFG_STM32_OTSL_SCMI_CLOCK_SUPPORT ?= y
+
+# When CFG_DT_CACHED_NODE_INFO is enabled, boot time may be divided by 2 or
+# more (depending on DTB size).
+CFG_DT_CACHED_NODE_INFO ?= y
-- 
2.47.2

